lib/bech32/src/Codec/Binary/Bech32/Internal.hs

```diff
@@ -293,27 +295,40 @@ humanReadableCharMaxBound = chr 126
                             Encoding & Decoding
 -------------------------------------------------------------------------------}
 
+-- | Like 'encode' but allows output to be longer than 90 characters. This isn't
+-- ideal as the error detection becomes worse as string get longer but still
+-- acceptable.
```

Comment:
Tiny typo in comment. This should probably read:

> "as the string gets longer, but it's still acceptable."

Reply:
Fixed.